cfme/common/vm.py

```diff
@@ -522,20 +521,19 @@ class VM(BaseVM):
     TO_RETIRE = None
 
     retire_form = Form(fields=[
-        ('date_retire', {
-            version.LOWEST: date_retire_element,
-            "5.5": AngularCalendarInput(
-                "retirement_date", "//label[contains(normalize-space(.), 'Retirement Date')]")}),
+        ('date_retire', AngularCalendarInput(
+            "retirement_date", "//label[contains(normalize-space(.), 'Retirement')]")),
```

Comment:
From my point of view, locator should be unique, because if something will be changed we can easily track this changes. 
So i'm okay with 'Retirement Date'  or h3 summary.